packages/cubit/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.2.0-dev.1
+
+- **BREAKING**: rename `Transition` to `CubitTransition`
```

Comment:
Out of curiosity, why?

Reply:
Still experimenting but I would like to eventually get to a point where there is complete interoperability between cubit and bloc and the Transition naming conflict is a bit annoying. Haven’t finalized anything yet though